LastWork(newWork)/Views/evaluations/evaluations.js

```diff
@@ -1,26 +1,30 @@
 function EvaluationsTableHeader(option) {
     const headings = [];
     option.headings.forEach((el) => {
-        const generatedEl = `<th>${el}</th>`;
+        const generatedEl = `<th class="details_button">${el}</th>`;
         headings.push(generatedEl);
     });
     const headingsEl = headings.join('');
 
     return `
     <tr style="background-color: #03a8f3; color: white; font-size: 150%">
     ${headingsEl}
-    <th></th>
+    <th class="details_button"></th>
+    <th style="width: 40px"></th>
     </tr>
     `;
 }
 
 function EvaluationsTableRow(option) {
     return `
-    <tr>
-        <td>${option.name}</td>
-        <td>${option.technology}</td>
-        <td>${option.level}</td>
-        <th>Detalii</th>
+    <tr class="details_button">
+        <td class="details_button">${option.CandName}</td>
+        <td class="details_button">${option.dateForm}</td>
+        <td class="details_button">${option.b}</td>
+        <th style="text-align: right; margin-right: 10px">Detalii</th>
+        <td id="detailsDivId"style="width: 40px">
```

Comment:
Will this mean that you can have multiple td elements with the same id?

Reply:
I had to use a class, but now I removed the id, i don;t need it anymore. Thanks ^_^